packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -200,6 +216,26 @@ export const NavigatorWrapper = () => {
           <AlertBanner />
           <InviteFriendModal isVisible={isInviteModalVisible} onInvite={onInvite} />
         </View>
+        <Modal isVisible={shakeForSupportVisible} backdropOpacity={0.5} style={styles.modal}>
+          <View style={styles.shakeForSupport}>
+            <Touchable
+              style={styles.closeButton}
+              onPress={onCancelSupport}
+              borderless={true}
+              hitSlop={variables.iconHitslop}
+            >
+              <Times />
+            </Touchable>
+            <Text style={styles.supportTitle}>{i18n.t('accountScreen10:havingTrouble')}</Text>
+            <Text style={styles.supportSubtitle}>{i18n.t('accountScreen10:shakeForSupport')}</Text>
+            <Button
+              onPress={onContactSupport}
+              text={i18n.t('accountScreen10:contactSupport')}
+              type={BtnTypes.PRIMARY}
+              testID="ContactSupportFromShake"
+            />
+          </View>
+        </Modal>
```

Comment:
Would be nice to extract this into its own component and add a simple unit test for it (checking tap triggers the right action?).